src/main/java/org/disq_bio/disq/HtsjdkReadsRddStorage.java

```diff
@@ -11,21 +11,41 @@
 import org.disq_bio.disq.impl.file.HadoopFileSystemWrapper;
 import org.disq_bio.disq.impl.file.NioFileSystemWrapper;
 import org.disq_bio.disq.impl.formats.bam.BamSink;
-import org.disq_bio.disq.impl.formats.bam.BamSource;
 import org.disq_bio.disq.impl.formats.cram.CramSink;
-import org.disq_bio.disq.impl.formats.cram.CramSource;
 import org.disq_bio.disq.impl.formats.sam.AbstractSamSink;
 import org.disq_bio.disq.impl.formats.sam.AbstractSamSource;
 import org.disq_bio.disq.impl.formats.sam.AnySamSinkMultiple;
 import org.disq_bio.disq.impl.formats.sam.SamFormat;
 import org.disq_bio.disq.impl.formats.sam.SamSink;
-import org.disq_bio.disq.impl.formats.sam.SamSource;
 
 /** The entry point for reading or writing a {@link HtsjdkReadsRdd}. */
 public class HtsjdkReadsRddStorage {
 
   /** An option for configuring how to write a {@link HtsjdkReadsRdd}. */
-  public interface WriteOption {}
+  public interface WriteOption {
+    static AbstractSamSink getSink(
```

Comment:
I thougth this more like a method for the `FormatWriteOption` instead of here, and an non-static method (see the proposal below).